jekyll/_cci2/gh-bb-integration.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: classic-docs
+title: GiHub and Bitbucket Integration
+description: GiHub and Bitbucket Integration
+categories: [migration]
+Order: 60
+---
+
+This document provides an overview of the process of checking out your code from GitHub or Bitbucket and running your configuration. After you add a project to CircleCI CircleCI adds the following GitHub or Bitbucket settings to your project using the permissions you gave CircleCI when you signed up:
```

Comment:
> After you add a project to CircleCI CircleCI adds the following GitHub or Bitbucket settings to your project using the permissions you gave CircleCI when you signed up

change to

"After you add a project to CircleCI, the following GitHub or Bitbucket settings are added to your project using the permissions you gave CircleCI when you signed up:"